portfolio/src/main/webapp/script.js

```diff
@@ -196,3 +203,15 @@ function createBigfootSightingsMap() {
     google.maps.event.trigger(map, "resize");
   });
 }
+
+function fetchBlobstoreUrlAndShowForm() {
+  fetch('/blobstore-upload-url')
+      .then((response) => {
+        return response.text();
+      })
+      .then((imageUploadUrl) => {
+        const messageForm = document.getElementById('my-form');
+        messageForm.action = imageUploadUrl;
+        messageForm.classList.remove('hidden');
+      });
+}
```

Comment:
just fyi, you could also use the javascript async-await syntax. sometimes makes the code consise.
Both these snippets are equivalent-
```suggestion
async function fetchBlobstoreUrlAndShowForm() {
  let uploadUrlResponse = await fetch('/blobstore-upload-url');
  
  const messageForm = document.getElementById('my-form');
  messageForm.action = uploadUrlResponse.text();
  messageForm.classList.remove('hidden');
}
```